desktop/app/app.go

```diff
@@ -99,6 +99,7 @@ type App struct {
 func NewApp(flags flashlight.Flags, configDir string, proClient proclient.ProClient, settings *settings.Settings) *App {
 	analyticsSession := newAnalyticsSession(settings)
 	app := &App{
+		Flags:                     flags,
```

Comment:
This was missing, causing the desktop client to completely disregard all flags.

Reply:
Great catch, thanks @hwh33